### PR TITLE
Added a long page table decision

### PR DESCRIPTION
**Pull request by joe513**
#31 fix

### CODE REVIEW

File: app/templatetags/extra_tags.py [hunk 1]
```diff
@@ -11,3 +11,24 @@ def url_replace(request, field, value):
     query_string[field] = value
 
     return query_string.urlencode()
+
+
+@register.simple_tag
```
Comment (inoks) [inline app/templatetags/extra_tags.py]: Can you please add tests?

Comment (joe513) [reply app/templatetags/extra_tags.py]: Yeah, I was gonna do this!

File: app/tests.py [hunk 2]
```diff
@@ -166,3 +168,28 @@ def test_url_replace_tags(self):
...
+    def test_ellipsis_or_number(self):
+        paginator = Paginator([x for x in range(1000)], 50)
+        page_count = paginator.num_pages
+
+        request = HttpRequest()
+
+        request.GET['page'] = 1
+        chosen_page = request.GET['page']
+
+        for page in range(1, page_count):
```
Comment (inoks) [inline app/tests.py]: This test is a bit complex currently, can you add some more specific use cases for the tests instead.
For example:
- there 11 pages and we on the 5 page right now
- there 11 pages and we on the 1 page right now
- there 11 pages and we on the 11 page right now
- there 3 pages and we on the 1 page right now
?

Comment (joe513) [reply app/tests.py]: Yes, I will try!

**Pull request closed without merging**